doc/nrf/getting_started/recommended_versions.rst

```diff
@@ -93,8 +93,13 @@ Not applicable
 .. note::
    The |NCS| tools are not supported by the older versions of the operating system.
 
+|NCS| toolchain
+***************
+
+The |NCS| toolchain is based on the toolchain of the Zephyr SDK (GNU ARM Embedded Toolchain) plus a set of tools specific to the |NCS|.
```

Comment:
Zephyr SDK is multi-arch, is `GNU ARM Embedded Toolchain` is https://developer.arm.com/downloads/-/gnu-rm which is something different, right?